teuthology/orchestra/run.py

```diff
@@ -221,6 +238,63 @@ def __repr__(self):
             name=self.hostname,
             )
 
+def find_unittest_error(xmlfile_path: str, client):
+    """
+    Parse the xml file.
+    If xmlfile_path is dir, parse all xml files.
+    """
+    if not xmlfile_path:
+        return "No XML file path was passed to process!"
```

Comment:
I think you might throw an exception here instead of returning a string since the caller of the function will receive a string and teuthology will use that as unitTestError message